bot/src/main.rs

```diff
@@ -598,7 +598,7 @@ fn get_config() -> BoxResult<(Config, RpcClient, Box<dyn GenericStakePool>)> {
                     .long("baseline-stake-amount")
                     .value_name("SOL")
                     .takes_value(true)
-                    .default_value("5000")
+                    .default_value("5")
```

Comment:
This is a safe default amount while we're working with small stakes

Reply:
Should we just take out the default value entirely?

Reply:
Yeah, it's always being set anyway